lib/rmagick_internal.rb

```diff
@@ -8,6 +8,18 @@
 #               to the classes.
 #==============================================================================
 
+if RUBY_PLATFORM =~ /mingw/i
+  begin
+    require 'ruby_installer'
+    ENV['PATH'].split(File::PATH_SEPARATOR).each do |path|
```

Comment:
Not sure if this is as readable, but we could do:

```rb
ENV['PATH'].split(File::PATH_SEPARATOR).grep(/ImageMagick/i).each do |path|
```

Reply:
I updated the code with your suggestion. Thank you :)